.gitignore

```diff
@@ -7,3 +7,4 @@ coverage*
 .DS_Store
 .env
 /env
+.coveragerc
```

Comment:
This should have been ignored, but it's showing up in this PR. Should we delete it?

Reply:
There is a merge conflict that needs to be resolved first @jordy1611 @Kathybui732

Reply:
Why is the CI failing?